yt/odin/requirements.txt

```diff
@@ -3,4 +3,8 @@ Flask==2.0.3
 Werkzeug==2.2.2
 cheroot==10.0.0
 
+# required by default checks
```

Comment:
What do you mean by "default" here?

Reply:
I mean checks which exist out-of-the-box and are deployed by `vendor` Docker image

Reply:
Discussed personally; by "vendor" Dima means the checks included in the default Odin package (in contrast to the private checks, that someone may possibly additionally introduce on top of this package).

Ok, I'd replace word "default" in this context, let's just simply write:
```
# Required by some of the checks.
```